pages/common/ab.md

```diff
@@ -10,10 +10,10 @@
 
 `ab -n 100 -c 10 {{url}}`
 
-- Use keep alive
+- Use keep alive:
 
 `ab -k {{url}}`
 
-- Use time limit. Represents the maximum number of seconds to spend for benchmarking.
+- Set the maximum number of seconds to spend for benchmarking:
 
-`ab -t 60 {{url}}`
\ No newline at end of file
+`ab -t 60 {{url}}`
```

Comment:
Perhaps `60` should be in tokens (`{{` and `}}`)?

Reply:
Yeah, same for the 100 and 10 above.